src/main/java/core/basesyntax/db/AllFruitInfo.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.FruitInfo;
+import java.util.ArrayList;
+import java.util.List;
+
+public class AllFruitInfo {
+    private List<FruitInfo> fruitInfoList = new ArrayList<>();

```

Comment:
you already have a fruitStorage, we don't need to store the list of transactions, we have on business logic for it. Get in hashmap is much faster